content/config/daemon/proxy.md

```diff
@@ -37,34 +52,30 @@ its start-up environment to configure HTTP or HTTPS proxy behavior:
 - `NO_PROXY`
 - `no_proxy`
 
-In Docker Engine version 23.0 and later versions, you may also configure proxy
-behavior for the daemon in the [`daemon.json` file](./index.md#configure-the-docker-daemon):
+Set any of these variables and restart the daemon for the proxy configuration
+to take effect:
 
-```json
-{
-  "proxies": {
-    "http-proxy": "http://proxy.example.com:3128",
-    "https-proxy": "https://proxy.example.com:3129",
-    "no-proxy": "*.test.example.com,.example.org,127.0.0.0/8"
-  }
-}
+```console
+$ export HTTP_PROXY=http://proxy.example.com:3128
+$ export HTTPS_PROXY=https://proxy.example.com:3129
+$ export NO_PROXY=localhost,127.0.0.1,docker-registry.example.com,.corp
+$ sudo systemctl restart docker
```

Comment:
Not sure if this works, because this requires the env-vars to be set in the systemd unit's environment, which is separate from the shell used to control systemd. So exporting these in the current environment would only work when running the actual `dockerd` binary from it (and when using `sudo`, you may loose those as well 😅 unless you use `-E`).

The only way to set these env-vars for a systemd service is through the systemd unit that starts it (or the `daemon.json`).

Reply:
I've updated to reflect this